docs-ref-services/servicebus.md

```diff
@@ -17,131 +17,77 @@ ms.service: service-bus
-scheduling, complex subscriptions, as well as workflow and transaction handling.
+Service Bus provides highly reliable queues and publish/subscribe topics with deep feature capabilities such as 
+ordered delivery, sessions, partitioning, scheduling, complex subscriptions, as well as workflow and transaction handling.
 
-The Service Bus feature capabilities are comparable and often exceed those of high-end, on-premises legacy message 
-brokers. The Service Bus features are available via standards-based protocols like AMQP 1.0 and HTTPS and all 
-protocol gestures are fully documented, allowing for broad interoperability. 
-
-Focusing on highly available and reliable durable messaging, the Service Bus Premium provides competitive throughput 
-performance even with substantial local datacenter deployments, but without hardware selection and acquisition 
-processes, deployment planning and execution, and endless performance optimization sessions. 
-
-Service Bus Premium is a fully managed offering with dedicated capacity reserved for each tenant that yields 
-predictable performance with a simple, capacity-oriented pricing model and at extremely lower overall cost than 
-commercial on-premises brokers. For many customers, Service Bus Premium can replace dedicated on-premises messaging 
-clusters today, even if the attached workloads do not run in the cloud. 
-
-Learn more about Service Bus concepts [in the messaging documentation section](https://docs.microsoft.com/azure/service-bus-messaging/) 
+Learn more about Service Bus concepts [in the messaging documentation section](https://docs.microsoft.com/azure/service-bus-messaging/). 
 
 For Java developers, Service Bus provides a Microsoft supported native API and Service Bus can also be used with
 AMQP 1.0 compliant libraries such as Apache Qpid Proton's JMS provider.
 
-## Client library
+## Libraries for resource management
 
-The official Service Bus client is available in [source code form on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/) and
-binaries and packaged sources [are available on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22azure-servicebus%22).
+Create and manage namespaces, topics, queues, and subscriptions with the management API.
 
-**The [sample code repository](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/) contains samples for:**
-* How to use the [QueueClient](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/azure-servicebus/QueuesGettingStarted/src/main/java/com/microsoft/azure/servicebus/samples/queuesgettingstarted/QueuesGettingStarted.java)
-* How to use the [TopicClient and SubscriptionClient](https://github.com/Azure/azure-service-bus/tree/master/samples/Java/azure-servicebus/TopicsGettingStarted/src/main/java/com/microsoft/azure/servicebus/samples/topicsgettingstarted)
+**Please find some examples here:**
```

Comment:
Instead of having this section, can we add a column to the table that follows for "Samples"?

Reply:
They are in separate repositories, And there are two links one for queue and second for topics. I am going to add these two links under Samples, like you said.